### DISCUSSION TITLE
Alerts lose color in light mode

### DISCUSSION THREAD

**Discussion by sergefdrv** [colinwilson/lotusdocs#39] [General]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

[Alert boxes](https://lotusdocs.dev/docs/guides/shortcodes/alerts/) lose their color in light mode.

### Expected behavior

Alert boxes should have similar color in both dark and light modes.

### Screenshots

![image](https://github.com/colinwilson/lotusdocs/assets/11386423/8a48069d-69f0-4a63-8353-ecb0fba59338)

### Environment

```markdown
- OS: Ubuntu
- Browser: Brave, Firefox
- Browser Version: Brave v1.57.53, Firefox 116.0.3
```

**Comment by colinwilson** [comment]
Thank you for the suggestion.

But at this moment in time I'd prefer to keep the pastel light mode colour scheme for alerts as it is. I'll take another look at this and the overall colour scheme again before the first release.

**Reply by colinwilson** [reply]
💡It may be an idea to offer a `bold` option in the config for users who prefer more assertive colours for their alerts?

**Reply by ghost** [reply]
Please don't change the alert colors - they're very good as they are. If you do plan on changing it, an additional option (like `bold`, etc.) would be better.

**Reply by sergefdrv** [reply]
Hmm, there must be something wrong with my monitor. Now on smartphone screen, it looks fine.

**Reply by colinwilson** [reply]
>  there must be something wrong with my monitor

@sergefdrv I think this is a valid point. There's a wide variance of default monitor settings + everyone has their preferred settings (warm, cold). I think I'll add a `bold` option (default `false`) in the `hugo.toml` config to accommodate a wider range of displays.

> Please don't change the alert colors - they're very good as they are

@saurabh-mish  Thanks. 👍
